### PR TITLE
make some tables scrollable

### PR DESCRIPTION
**Pull request by lcarcone**
Hi Tripu,

I made small changes to make some large tables scrollable (questions 13 and 15).
It is not complete as I'd like the first column to be fixed so that it would be easy to scroll the table horizontally, but I haven't yet found a solution to do that without js.
Maybe you know a css solution, otherwise we can ask Bert.
What do you think ?

Thanks,
Laurent

### CODE REVIEW

**Comment by lcarcone** [comment]
Hi Tripu,

I adapted a css tip from Bert to have a scrollable table with fixed first colum.
It's specific to questions 13 and 15 for the moment, there are still some work to do if we want to generalize it to all tables.
I had to override a rule of the w3c site css to have this solution to work, I have to check it doesn't break something else.

I have also tested 2 jquery plugins: dataTables and fixedColumns. 
I had some problems to use fixedColumns with our examples.
DataTables can help us to do vertical scrolling with fixed header (I don't know if there is a css tip to do that). It is quite powerful to manage tables, maybe we can look at it to improve the usability of results page.

Thanks,
Laurent

**Review by tripu — CHANGES_REQUESTED** [review]
@lcarcone, :+1:!

A couple of suggestions.

Perhaps you don't need to wrap tables on a new `div`; instead, add the necessary class (`scroll`) to the existing `table`.

Complex table headings (when there is more than one `tr` inside `thead`) don't seem to behave well when scrolling. I tried to tweak properties of several elements in CSS to anchor that first column better, but I don't like results either. I wonder if it won't be better to leave all columns with their default `position: static`, even if the first column isn't visible when the user scrolls&hellip;

**Comment by tripu** [comment]
(This fixes #28.)

**Comment by lcarcone** [comment]
@tripu
I made several tests, the class (scroll) must be on the parent wrapper, it doesn't work on the table itself.
I also made several tests to have the first header column fixed but without success. I removed the property 'position: absolute;' on the  first date cell for the moment, I'll do more tests when it is merged.
Thanks

File: css/site-wbs-style.css [hunk 1]
```diff
@@ -112,7 +112,7 @@ div.scroll tr th:first-child, div.scroll tr td:first-child {
 }
 
 div.scroll tr td:first-child {
-   position: absolute;
+   // position: absolute;
```
Comment (tripu) [inline css/site-wbs-style.css]: Use `/* */` instead (but I'd say you can remove the property altogether).

[`w3.org/TR/CSS2`](https://www.w3.org/TR/CSS2/syndata.html#comments)
[`w3.org/TR/css-syntax-3`](https://www.w3.org/TR/css-syntax-3/#comment-diagram)

**Review by tripu — CHANGES_REQUESTED** [review]
:+1:, barring my comment about comments :)

**Comment by lcarcone** [comment]
I removed the property, thanks

**Review by tripu — APPROVED** [review]

**Pull request merged**